oid4vci/integration/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-slim-bookworm AS base
+FROM python:3.10-slim-bullseye AS base
```

Comment:
`bookworm` => `bullseye`, is a downgrade from Debian 12 to Debian 11.  Is this intentional?

Reply:
No. Not intentional. Just needs to be 3.10. Updated.